Apps/Common/src/Delegates/ClientRegistriesDelegate.cs

```diff
@@ -378,26 +378,15 @@ private bool PopulateNames(HCIM_IN_GetDemographicsResponseIdentifiedPerson retri
                 }
             }
 
-            PN nameSection = (documentedName ?? legalName)!;
```

Comment:
Isn't ClientRegistriesDelegete in common pointing to v1?  Isn't v1 supposed to be left untouched?